frontend/controller/actions/identity.js

```diff
@@ -21,6 +21,22 @@ import type { Key } from '../../../shared/domains/chelonia/crypto.js'
 import { handleFetchResult } from '../utils/misc.js'
 import { encryptedAction } from './utils.js'
 
+const requireToSuccess = async (fnToSuccess) => {
+  if (typeof fnToSuccess !== 'function') {
+    return
+  }
+
+  let failureCount = 0
+  do {
+    try {
+      return await fnToSuccess()
+    } catch (err) {
+      console.log('[requireToSuccess:]', err)
+      failureCount++
+    }
+  } while (failureCount < 10)
+}
```

Comment:
I think @taoeffect or @corrideat could recommend the better way to handle this.

Reply:
What is the reason for this function? It seems unnecessary to me. What happens if we remove it?

Reply:
This is similar to how we do when publish events. As you know, if it fails to publish an event, we should try again and again to make it succeed. Similar to the above case, it could fail to save the value to KV store sometimes. So, we retry to make sure that the value to be saved in KV store.

> What happens if we remove it?

It's same to skip an event not to be published.